config_defaults/subtests/docker_cli/run.ini

```diff
@@ -85,3 +75,29 @@ docker_timeout = 60
 remote_image_fqin = stackbrew/centos:7
 run_options_csv =
 cmd = /bin/true
+
+[docker_cli/run/run_install]
+generate_generic = yes
+__example__ = base_repo_name
+run_options_csv = -i,--rm
+#: Repo./Channel name to enable for yum commands
+base_repo_name = base
+#: Packages to install
+install_rpms = iputils
+#: Command to run to install rpms (yum or dnf)
+install_cmd = yum --disablerepo "*" --enablerepo "%(base_repo_name)s" install -y %(install_rpms)s
```

Comment:
Tried that, won't work b/c then ``install_cmd`` and ``cmd`` will be under different quoting.  Instead, I removed all quoting, and just do it in the test-code instead.  If that doesn't work...then we can come up with something else.